release_draft.md

```diff
@@ -0,0 +1,305 @@
+#### Table of Contents
+
+[Triggering a Release]()
+  * [Prerequisites]()
+     - [Software and Configurations]()
+     - [Credentials]()
+  * [Preparing Ignition Software](
+     - [Update version]()
+     - [Update binary version]()
+  * [Launch the release in the building server]()
+    - [Executing release.py]()
+      - [dry-run simulation mode]()
+      - [release.py for prerelease or nightlies]()
+      - [release.py for revision bumps]()
+
+
+## Triggering a Release
+
+### Understanding the releasing process
+
+The following picture shows the interactions triggered when using the release
+tool `release.py` explained in this guide:
+
+![release.py interactions](releasing/images/releasepy_execution.svg)
+
+Actions for releasing a new version of library `foo` with major version `X`:
+
+ 1. `release.py` will generate a local tarball with the source code of the new
```

Comment:
It would be interesting to add links to most of this stuff. The `release.py` script, the various jobs in build.osrfoundation.org...

Reply:
agree de2a3997b9d651cddf8f6fc9611aa38dcf9c3efd